worker/src/handlers/post.ts

```diff
@@ -95,7 +96,7 @@ const sanitizePayload = (
     throw new Error("Missing required keys in the payload");
   }
 
-  if (!InstallationTypes.includes(payload.installation_type)) {
+  if (!Object.keys(InstallationTypes).includes(payload.installation_type)) {
```

Comment:
This is faster as it avoids generating an array.
```suggestion
  if (!(payload.installation_type in InstallationTypes)) {
```